docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
+
+- Provide feedback on proposals or ideas
+- Discuss possible use-cases or feature requests
+- Make suggestions for non-technical contributions, including events, writing, or business models
+- Answer questions about the protocol, services and products.
+
+
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
```

Comment:
Again, would add periods to align with the other bullet point
```suggestion
 - Open an issue on the corresponding repository.
 - Give your request a clear title.
 - Provide a written outline of the feature request for discussion.
```

Reply:
@Dudleyneedham According to your statement above, you have to move the period to the last bullet point then.